### PR TITLE
Adds a surrender button for when you are being held at gunpoint

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request

An alert that gives you the opportunity to quickly do the *surrender emote by clicking on it. Currently used for being held at gunpoint but could be used for other things. This adds little mechanically but serves as an easy way for people to surrender.

## Why It's Good For The Game
Assistance for new players and convinces for veterans.

## Changelog
:cl: itseasytosee
add: Adds a surrender button that pops up when you are at gunpoint that can be pressed to do the *surrender emote
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
Hey that's pretty cool. It probably should give some message of some kind though, I think. Raising your hands in the air (or if you don't have them, then 'X suddenly freezes in place, seemingly to surrender.'

**Comment by Bawhoppen** [comment]
Oh woops. Ignore my deleted comment. I thought you were adding surrender emote. Good PR either way :+1:

**Comment by maxymax13** [comment]
why is the icon a small little man with no legs

**Comment by ShibaInuLord** [comment]
> 
> 
> why is the icon a small little man with no legs

its a man raising a white flag

**Comment by Pugie** [comment]
> > why is the icon a small little man with no legs
> 
> its a man raising a white flag

make it mime only

File: code/datums/status_effects/neutral.dm [hunk 1]
```diff
@@ -293,6 +301,23 @@
...
+/datum/status_effect/surrender
+	id = "surrender"
+	duration = -1
+	tick_interval = -1
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/surrender
+
+/atom/movable/screen/alert/status_effect/surrender
+	name = "Surrender"
+	desc = "Looks like your in trouble now, bud. \ Click here to surrender (Warning: you will be paralyzed for 20 seconds)"
```
Comment (Jared-Fogle) [inline code/datums/status_effects/neutral.dm:314]:
```suggestion
	desc = "Looks like you're in trouble now, bud. \ Click here to surrender (Warning: you will be paralyzed)."
```

I have removed the timer as I don't think it's important and don't want two places to have to update.

Also, what's the backslash for? This looks like it's just going to create a second space.

Comment (itseasytosee) [reply code/datums/status_effects/neutral.dm:314]: yeah sorry, I've been off of beyond for a while now and I instinctually went to make a new line after the sentence. It's unimportant and Ill remove it.

File: code/datums/status_effects/neutral.dm [hunk 2]
```diff
@@ -293,6 +301,23 @@
...
+/datum/status_effect/surrender
+	id = "surrender"
+	duration = -1
+	tick_interval = -1
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/surrender
+
+/atom/movable/screen/alert/status_effect/surrender
+	name = "Surrender"
+	desc = "Looks like you're in trouble now, bud. Click here to surrender (Warning: You will be incapacitated)"
```
Comment (cacogen) [inline code/datums/status_effects/neutral.dm:314]:
```suggestion
	desc = "Looks like you're in trouble now, bud. Click here to surrender. (Warning: You will be incapacitated.)"
```

**Review by Mothblocks — APPROVED** [review]
Codes fine, Twaticus please merge when the art is okay.

**Pull request merged**